### PR TITLE
Workshop venue information

### PR DESCRIPTION
**Pull request by marcoow**
Bildschirmfoto 2023-08-16 um 15 38 00

### CODE REVIEW

File: templates/partials/workshop-venue.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <h3><a name="workshop-venue">Workshops Venue</a></h3>
+  <div class="image-venue-container">
+    <picture>
+      <source srcset="/images/workshop-venue/mobile/comet-louise.webp" media="(max-width: 640px)" />
+      <img class="image-venue" src="/images/workshop-venue/comet-louise.webp" alt="workshop venue picture" />
+    </picture>
+  </div>
+
+  <p>The EuroRust workshops will take place in the Comet Louise coworking space in the city center of Brussels. The venue is located at Pl. Stéphanie 20, 1050 Brussels.</p>
+  <a class="map-link" href="https://goo.gl/maps/h8Kmv5Bi356mUMnc6" alt="Google Maps for conference venue address"
```
Comment (oscard0m) [inline templates/partials/workshop-venue.html:11]: ⚠️ Wrong Google Maps link?

```suggestion
  <a class="map-link" href="https://goo.gl/maps/vZEUrH9EMzHDSW256" alt="Google Maps for workshop venue address"
```

**Review by oscard0m — COMMENTED** [review]
LGTM

Check Google Maps link

**Comment by oscard0m** [comment]
nitpick:

In very small viewport the button for Google Maps breaks:

image

Feel free to open a GitHub issue to address this in a different PR

**Review by oscard0m — APPROVED** [review]

**Comment by marcoow** [comment]
Hm, yeah the issue with the maps link in very small viewports happens for the main conference venue part as well – we'll see whether someone finds some time to fix that eventually…

**Pull request merged**